### PR TITLE
Release notes for 6.0.0-rc.4 5.6.4 5.5.12 5.2.13

### PR DESCRIPTION
**Pull request by knisbet**
Release notes for:
* 6.0.0-rc.4
* 5.6.4
* 5.5.12
* 5.2.13

### CODE REVIEW

File: docs/5.x/changelog.md [hunk 1]
```diff
@@ -31,6 +31,21 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 6.x Releases
 
+### 6.0.0-rc.4
+
+#### Bugfixes
+
+* Fix a security issue with insecure decompression of application bundles
```
Comment (a-palchikov) [inline docs/5.x/changelog.md]: nit: missing dot at the end.

File: docs/5.x/changelog.md [hunk 2]
```diff
@@ -31,6 +31,21 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
...
+* Fix a security issue that allowed remote code execution in the tele cli tool.
+* Fix a security issue with missing ACLs in internal API.
+* Fix a security issue with install scripts command injection.
+* Fix a security issue that allowed for two factor authentication to be bypassed.
+* Fix a security issue that allowed for cross site scripting in Internet Explorer.
```
Comment (a-palchikov) [inline docs/5.x/changelog.md]: nit: cross-site

File: docs/5.x/changelog.md [hunk 3]
```diff
@@ -82,6 +97,51 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
...
+
+!!! warning
+    This release fixes security vulnerabilities within Gravity. Please see
+    [Gravity Enterprise Announcement](https://gravitational.zendesk.com/hc/en-us/articles/360025697553-Gravity-Enterprise-6-0-0-rc-4-5-6-4-5-5-12-5-2-13-Security-Update) for more information.
```
Comment (a-palchikov) [inline docs/5.x/changelog.md]: Announcement URL requires a login - is this intended?

Comment (knisbet) [reply docs/5.x/changelog.md]: I'll make the announcement public when I publish the release notes. Thanks for double checking :)

**Pull request merged**